app/design/frontend/Clothes/touch/Magento_Theme/page_layout/2columns-extra-list.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<layout xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_layout.xsd">
+    <update handle="2columns-left"/>
```

Comment:
@yuratouch According to the ticket requierments, handle should be 2columns-right: `<update handle="2columns-right"/>`